example/src/Examples/Graphs/Interpolation.tsx

```diff
@@ -25,28 +23,18 @@ export const Interpolation: React.FC<GraphProps> = ({ height, width }) => {
     createGraphPath(width, height, 60)
   );
 
-  const onPress = useCallback(() => {
-    setCurrentPath(createGraphPath(width, height, 60));
-  }, [height, width]);
-
   useEffect(() => {
-    const isRunning = { value: true };
-    const dispatchChange = () => {
-      setTimeout(() => {
-        setCurrentPath(createGraphPath(width, height, 60));
-        if (isRunning.value) {
-          dispatchChange();
-        }
-      }, Math.random() * 400 + 500);
```

Comment:
Sorry for not bringing this up earlier, but setInterval does not need `Math.random() * 400 + 500` since it will only be evaluated once - right?